CHANGES.txt

```diff
@@ -4,7 +4,9 @@ Changes for Crate
 
 Unreleased
 ==========
- 
+
+ - Fixed division by zero which returns now an `ArithmeticException`.
```

Comment:
- Division by aggregation that resulted in 0 (e.g. ``SELECT max(long_col) / min(long_col) from t``) now correctly returns ``ArtithmeticException`` and does not hang any more.

or something similar